CHANGELOG.md

```diff
@@ -26,6 +26,10 @@
 - **Feature:** Added sample implementation for Sharing the message from Mini app
 - **Feature:** Added sample app implementation to revoke/manage the custom permissions for the list of downloaded mini apps
 
+### 2.x
+
+***SDK***
+-**Fixed:** support of `playsinline` instruction of  `Video` html tag
```

Comment:
I think this has to be moved to the top `2.x.0`. That has our next release updates

Reply:
Sorry, this should be move to a separate release version above the other `2.x.0` - this change won't be included in the `2.1.0` release.

Reply:
men! I thought I was safe with 2.x as it's just not even a version number XD